src/data/markdown/docs/30 xk6-browser/02 API/03 BrowserType/launch--options--.md

```diff
@@ -13,12 +13,58 @@ Launches a new browser process.
 | slowMo            | string   | `null`  | Slow down input actions and navigation by the specified time e.g. `'500ms'`.                                                                                                                                                                                          |
 | timeout           | string   | `'30s'` | Default timeout to use for various actions and navigation.                                                                                                                                                                                                            |
 
 
+### Default arguments
+
+List of default arguments included when launching the browser process.
+
+**Note:** The starting '--' have been omitted from the argument names in these lists.
+
+| Argument | Value  | Description                                            |
+|----------|--------|--------------------------------------------------------|
+| disable-background-networking | `true` | Disables several subsystems which run network requests in the background. This is for use when doing network performance testing to avoid noise in the measurements. |
+| enable-features | NetworkService,<br />NetworkServiceInProcess | Comma-separated list of feature names to enable. |
+| disable-background-timer-throttling | `true` | Disables task throttling of timer tasks from background pages. |
+| disable-backgrounding-occluded-windows | `true` | Disables backgrounding renders for occluded windows. Done for tests to avoid nondeterministic behavior. |
+| disable-breakpad | `true` | Disables the crash reporting. |
+| disable-component-extensions<br />-with-background-pages | `true` | Disables default component extensions with background pages. Useful for performance tests where these pages may interfere with results. |
+| disable-default-apps | `true` | Disables installation of default apps on first run. This is used during automated testing. |
+| disable-dev-shm-usage | `true` | The /dev/shm partition is too small in certain VM environments, causing Chrome to fail or crash. This flag provides a work-around for this issue (a temporary directory will always be used to create anonymous shared memory files). |
+| disable-extensions | `true` | Disables extensions. |
+| disable-features | ImprovedCookieControls,<br />LazyFrameLoading,<br />GlobalMediaControls,<br />DestroyProfileOnBrowserClose,<br />MediaRouter,<br />AcceptCHFrame | Comma-separated list of feature names to disable. |
+| disable-hang-monitor | `true` | Suppresses hang monitor dialogs in renderer processes. This may allow slow unload handlers on a page to prevent the tab from closing, but the Task Manager can be used to terminate the offending process in this case. |
```

Comment:
```suggestion
| disable-hang-monitor | `true` | Suppresses hang monitor dialogs in renderer processes. This may allow slowing unload handlers on a page to prevent the tab from closing, but the Task Manager can be used to terminate the offending process in this case. |
```

Reply:
I don't think this has exactly the same meaning no?

Reply:
Sounded more grammatically correct to me 🤷 😅

Reply:
1. This may allow slow unload handlers -- the unload handlers are slow
2. This may allow slowing unload handlers -- the unload handlers are slowing down

Which one is it?

Reply:
Yes, that's the subtle difference that I was understanding too. I would stick to number 1 as that's the one given in chromium source.